Modules/Forum/classes/class.ilForumTopic.php

```diff
@@ -258,9 +258,9 @@ public function getFirstPostId()
             '
 			SELECT * FROM frm_posts_tree 
 			WHERE thr_fk = %s
-			AND parent_pos = %s',
+			AND parent_pos != %s',
             array('integer', 'integer'),
-            array($this->id, '1')
+            array($this->id, '0')
```

Comment:
Hmmm, IMO we have to use the `depth` and `rgt` (`ORDER BY rgt DESC`) field in the condition and we should use a `$this->db->setLimit(1);`